compileAndBuildVLCKit.sh

```diff
@@ -547,10 +546,7 @@ buildLibVLC() {
     demuxdump
     fingerprinter
     output_udp
-    output_http
     output_livehttp
-    libmux
-    stream_out
```

Comment:
Please add more precise removal rules instead. We definitely don't need all stream_out and all muxing modules.